src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -234,6 +236,10 @@ def remove_dialog_tag(text)
     link_to(text, '#', title: 'Close', id: 'remove_dialog')
   end
 
+  def close_dialog_tag(text, target)
+    link_to(text, '#', title: 'Close', class: 'close_dialog', data: { target: target })
```

Comment:
@mschnitzer My reasoning here to not do this would be that we only use it once. So it's not worth to have this kind of indirection. Having the link right in the code means that it's easier to read it.